src/common/actions/repository-input.js

```diff
@@ -0,0 +1,69 @@
+import 'isomorphic-fetch';
+import parseGitHubUrl from 'parse-github-url';
+
+export const CHANGE_REPOSITORY_INPUT = 'CHANGE_REPOSITORY_INPUT';
```

Comment:
Perhaps a bikeshed, but the action is not to change the repository input, it's to change the repository url (or 'set', but my argument is against the word 'input', not the word 'change').